extensions/cornerstone/src/commandsModule.ts

```diff
@@ -68,16 +92,151 @@ const commandsModule = ({ servicesManager }) => {
       toolGroupIdToUse = toolGroup.id;
     }
 
-    const toolGroup = ToolGroupService.getToolGroup(toolGroupIdToUse);
+    const toolGroup = toolGroupService.getToolGroup(toolGroupIdToUse);
     return toolGroup;
   }
 
+  function _showViewerContextMenu(viewerElement, options) {
+    let defaultPointsPosition = [];
+    if (options.nearbyToolData) {
+      defaultPointsPosition = commandsManager.runCommand(
+        'getToolDataActiveCanvasPoints',
+        { toolData: options.nearbyToolData }
+      );
+    }
+
+    contextMenuController.showContextMenu(
+      options,
+      viewerElement,
+      defaultPointsPosition
+    );
+  }
+
   const actions = {
+    /** Show the specified context menu */
+    showViewerContextMenu: providedOptions => {
+      const viewerElement = _getActiveEnabledElement();
+
+      const options = { ...providedOptions };
+      const { event: evt } = options;
+      const { useSelectedAnnotation, nearbyToolData, menuName } = options;
+
+      if (menuName) {
+        Object.assign(
+          options,
+          customizationService.getModeCustomization(
+            menuName,
+            defaultContextMenu
+          )
+        );
+      }
+
+      if (useSelectedAnnotation && !nearbyToolData) {
+        const firstAnnotationSelected = getFirstAnnotationSelected(
```

Comment:
where is this getFirstAnnotationSelected imported?

Reply:
Added the import